crates/compiler/load/tests/test_reporting.rs

```diff
@@ -5018,7 +5028,7 @@ mod test_reporting {
     I was partway through parsing an `import`, but I got stuck here:
 
     4│      import svg.Path a
-                            ^
+                           ^
 
     I was expecting to see the `as` keyword, like:
```

Comment:
Not a blocker, but given that the error is now pointing to the space instead of the `a`, maybe this should now say `I was expecting to see the `as` keyword next, like:`